e2e-tests/cypress/tests/integration/channels/channel/convert_group_message_to_private_spec.js

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @channels @channel
+
+describe('Group Message', () => {
```

Comment:
Wonder if we want to add tests for the "receiving part" of the action. i.e. Have a user be in the GM, convert it directly through the API, and make sure the user is properly navigated, the channel disappear from the categories, etc...

Reply:
Done.